test/tests/functional/pbs_Rrecord_resources_used.py

```diff
@@ -82,13 +82,13 @@ def setUp(self):
         a = {'resources_available.ncpus': 4}
         rv = self.server.manager(MGR_CMD_SET, NODE, a,
                                  id=self.hostA, expect=True)
-        self.assertTrue(rv)
+        self.assertEqual(rv, 0)
```

Comment:
These are probably old tests.  You don't need to check the return value of manager().  It will raise an exception if it fails.